modules/navigation/godot_navigation_server.h

```diff
@@ -96,6 +96,8 @@ class GodotNavigationServer : public NavigationServer3D {
 	int pm_edge_connection_count = 0;
 	int pm_edge_free_count = 0;
 
+	const Vector<Vector3> dummy_empty_vertices;
```

Comment:
Addition of this so that `obstacle_get_vertices` is able to return by const ref (to avoid returning a dangling ref)

Reply:
I think we should remove the ref part same as with 2D, the servers usually do not return anything by ref.

Reply:
Can now be removed imo.